qbx_consumables/server.lua

```diff
@@ -1,17 +1,49 @@
 local config = require 'qbx_consumables.config'
 
-local function addHunger(amount)
+--- sets player nutrition level in statebag
```

Comment:
This is setting the state for `hunger`, probably best to use that nomenclature rather than `nutrition` in the docstring